domains/kelvinet.is-cool.dev.json

```diff
@@ -0,0 +1,23 @@
+{
+    "description": "Project Description",
+
+    "domain": "is-cool.dev",
+    "subdomain": "kelvinet",
+
+    "owner": {
+        "repo": "https://github.com/username/repo",
+        "email": "hello@example.com"
+    },
+
+    "record": {
+        "A": ["216.239.32.21", "216.239.34.21", "216.239.36.21", "216.239.38.21"],
+        "AAAA": ["::1", "::2"],
+        "CNAME": "ghs.google.com",
```

Comment:
CNAME cannot be added with other records.

Multiple CNAME records are not supported.